monkestation/code/modules/mob/living/carbon/life.dm

```diff
@@ -0,0 +1,2 @@
+/mob/living/carbon
+	var/bloodcrawl_allow_items = FALSE
```

Comment:
might be better as a trait, like how there's `VENTCRAWLER_NUDE` and `VENTCRAWLER_ALWAYS` - perhaps, `BLOODCRAWL_WITH_ITEMS`?

Reply:
Yeah that sounds good, I'll get to work on that